portfolio/src/main/webapp/script.js

```diff
@@ -30,13 +30,27 @@ function addRandomGreeting() {
 /**
  * Fetches a welcome message from the server and adds it to the DOM.
  */
-function getWelcomeMessage() {
+function getComments(selectObject) {
+  var lang = selectObject.value;
 
+  var english = "en";
+  var french = "fr";
+  var spanish = "es"
+  
   // The fetch() function returns a Promise because the request is asynchronous.
-  const responsePromise = fetch('/data');
-
+  if(lang == english){
+    const responsePromise = fetch('/data?lang=en');
+    responsePromise.then(handleResponse);
+  }
+  else if(lang == french){
+    const responsePromise = fetch('/data?lang=fr');
+    responsePromise.then(handleResponse);
+  }
+  else if(lang == spanish){
```

Comment:
Whether you refactor to use switch or keep if-else, you want to have a default/else statement here that handles input that is not English, French, or Spanish. Although it's not possible for your user to select any other language right now, this could lead to bugs later on as your codebase develops.

Reply:
Good idea! I made it display an message for now!